components/clarinet-deployments/src/types.rs

```diff
@@ -358,8 +401,10 @@ impl ContractCallSpecification {
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
```

Comment:
Note: We're safe to change ser/de rules for this struct even though the derive functions already existed. We verified by deleting the derives and confirming that everything compiled.